components/.storybook/main.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
This one could be a typescript file, what do you think? Like this [one](https://github.com/vtex/styleguide/blob/master/.storybook/main.ts)

If you need any help to configure typescript, ping us on discord :smile: